package.json

```diff
@@ -30,13 +30,13 @@
     "/index.css"
   ],
   "dependencies": {
-    "lit-html": "^1.1.1"
+    "lit-html": "^1.2.1"
```

Comment:
This is they key change. We want access to some new directives from lit-html in a related PR.